pkg/networkmanager/index.html

```diff
@@ -582,7 +582,7 @@ <h1 id="testing-connection-curtain-restoring" translatable="yes">Restoring conne
         </table>
       </div>
     </div>
-    <div class="panel panel-default" id="network-interface-slaves">
+    <div class="panel panel-default" id="network-interface-slaves" hidden>
```

Comment:
This only has effect initially; once the user has navigated between different interfaces, he/she will still see the old buggy behavior.

I think this should do it:

```
diff --git a/pkg/networkmanager/interfaces.js b/pkg/networkmanager/interfaces.js
index a90bde614..fa32dc710 100644
--- a/pkg/networkmanager/interfaces.js
+++ b/pkg/networkmanager/interfaces.js
@@ -2862,7 +2862,6 @@ PageNetworkInterface.prototype = {
             if (!con || (cs.type != "bond" && cs.type != "team" && cs.type != "bridge")) {
                 self.rx_series.add_instance(self.dev_name);
                 self.tx_series.add_instance(self.dev_name);
-                $('#network-interface-slaves').hide();
                 return;
             }
 
@@ -2997,6 +2996,7 @@ PageNetworkInterface.prototype = {
             update_network_privileged();
         }
 
+        $('#network-interface-slaves').hide();
         if (self.main_connection)
             update_connection_slaves(self.main_connection);
     }
```